itunes-conversion-input-xsd/src/main/resources/xsd/chapters/itunes-chapters.xsd

```diff
@@ -22,30 +22,26 @@
 <xs:schema attributeFormDefault="unqualified" elementFormDefault="qualified"
```

Comment:
We should not depend from generated iTunes metadata schema types. Please create missing types for custom chapters schema (can be copy-paste from generated metadata xsd).